src/fixtures/HelperFixtures.ts

```diff
@@ -68,7 +68,7 @@ export const test = base.extend<NonNullable<unknown>, FixtureTypes>({
             const config = (await response.json()) as { version: string };
 
             const features: FeaturesType = {};
-            if (satisfies(config.version, '>=6.6.0.0')) {
+            if (satisfies(config.version, '>=6.6.1.0')) {
```

Comment:
@pweyck I might be overlooking something, but it looks like the `/_action/feature-flag` route was introduced with `v6.6.1.0` and is not present in `v6.6.0.0`.

Reply:
Not sure if I break something by increasing this, but it will fail otherwise when executed with `v6.6.0.0`.